### PR TITLE
Add a feature flag provider

### PR DESCRIPTION
**Pull request by rebeccafitzsimmons1**
# What
Adds a feature flag provider.

# Why
So that we can deploy features without releasing them, and are able to toggle features on per environment.

# Notes
The feature flag provider takes an argument that corresponds to the name of the feature flag environment variable for the feature, e.g:

```
if (featureIsEnabled(`FEATURE_FLAG_TEST_FEATURE`)) {
    .....
}
```

where `featureIsEnabled` is the feature flag provider and, for the sake of this example, `FEATURE_FLAG_TEST_FEATURE` is the name of the env var for the 'test feature' feature flag. The feature flag env var needs to be set to `true` for the feature to be toggled on, e.g.

`FEATURE_FLAG_TEST_FEATURE=true`

and toggled to false, or simply omitted as an env var altogether, if the feature is to be toggled off.

### CODE REVIEW

File: src/helpers/featureFlag.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const featureIsEnabled = (feature) => {
+  return process.env[feature] === "yes" ? true : false;
```
Comment (MrTortoise) [inline src/helpers/featureFlag.js:2]: maybe "yes" and no could be true and false?

**Review by MrTortoise — COMMENTED** [review]
I think true and false is better ... and yeah that makes it even less 👍

File: src/helpers/featureFlag.js (hunk 1 above)
Comment (faissalMT) [inline src/helpers/featureFlag.js:2]:
```suggestion
  return process.env[feature] === "yes";
```
`===` already produces a boolean so the `?` is unnecessary

Comment (rebeccafitzsimmons1) [reply src/helpers/featureFlag.js:2]: lool monday brain 🤦‍♀️

**Pull request merged**